app/components/application/AppEndpointCard/AppEndpointCard.tsx

```diff
@@ -159,6 +163,73 @@ export default function AppEndpointCard({ app, blockchains }: AppEndpointProps)
               />
             )
           })}
+        <Grid
+          align="start"
+          justify="space-between"
+          sx={{
+            margin: "2em 0 0",
+            paddingTop: "2em",
+            borderTop: "1px solid #2F373E",
+          }}
+        >
+          <Box
+            sx={{
+              width: "100%",
+            }}
+          >
+            <Grid
+              align="center"
+              justify="space-between"
+              sx={{
+                margin: "0 0 2em 0",
+              }}
+            >
+              <Grid align="center" m={0}>
+                <Text m="0 0.5em 0 0" size="lg" weight={600}>
+                  Your Data API
+                </Text>
+                <Text m="0 0.5em 0 0" size="sm" weight={400}>
+                  (powered by Covalent)
+                </Text>
+                <HelpTooltip
+                  label={"This endpoint supports Ethereum, Polygon, BSC and more."}
+                />
+              </Grid>
+              <Anchor href="https://www.covalenthq.com/docs/api/" target="_blank">
+                <Button className="pokt-button-outline" color="blue" variant="outline">
+                  <Text fw="normal" fz="sm" mr="0.5em">
+                    Data API Docs
+                  </Text>
+                  <ExternalArrow fill={theme?.colors?.blue[5]} />
```

Comment:
There is an IconArrowUpRight from pocket-blocks that would be better here. We really sure just be removing those old icons still in this repo.